### DISCUSSION TITLE
Weird framerate when switching from song select to main menu

### DISCUSSION THREAD

**Discussion by Tinchozz** [ppy/osu#15611] [Q&A]
[video describing the issue](https://youtu.be/1MQdB1Bv-TU)

**Comment by bdach** [comment]
how many maps do you have imported?

**Reply by Tinchozz** [reply]
idk if there's a way to see that in lazer and i deleted my stable installation, but it probably is in the 5 digit range? (my folder is 72gb with all bg img/videos removed if that's anything to go by)

**Reply by peppy** [reply]
Most definitely from song select preloading in the background. Eventual plan is to do that less, via screen resuming or just optimisations post-realm.
